app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,78 +1,60 @@
-import React, { useState } from "react";
+import React from "react";
 
-import { Formik, Form } from "formik";
-import { Button, Pane } from "neetoui/v2";
-import { Input, Textarea } from "neetoui/v2/formik";
+import { Button, Pane, Input, Select, Toastr } from "neetoui/v2";
 
-import notesApi from "apis/notes";
-import formValidationSchemas from "constants/formValidationSchemas";
+import { TAGS as tags } from "./constants";
 
-export default function NoteForm({ onClose, refetch, note, isEdit }) {
-  const [submitted, setSubmitted] = useState(false);
-  const handleSubmit = async values => {
+export default function NoteForm({ onClose }) {
```

Comment:
Please rename the component to something more semantically correct. Also, we don't need to deep nest the file inside folders like `Pane` or `Form`. We can just create a file `Notes/Create.jsx`.

Reply:
Also, the `validationSchema` and `initialValues` should be moved to the scope of this directory. We should only place constants at a global level if and only if the constant is being reused at multiple places. Else, we should always keep it scoped to the feature or directory.

Reply:
@karthiknmenon I am not using formik since there's no actual submissions involved. Is that fine? Can I just omit the validation schemas then?

Reply:
Please use Formik. The validations would work even if you don't have an API wired.